### DISCUSSION TITLE
Clearify the feature: Manage pure SQL. Works with existing SQL databases, or helps build new architectures from scratch.

### DISCUSSION THREAD

**Discussion by highkay** [directus/directus#11548] [Archive]
If I am not misunderstanding this feature is used in a brownfield project. Like create a new item in directus to ref an exist record in legency database(the same db). But I cannot find any guide in docs to use this feature or I just should write extension to implement it myself?

**Comment by benhaynes** [comment]
Not sure I completely understand, but Directus is more like a database administration tool... it provides direct access to the database schema/content. So there's no real difference between new/existing schema/content. Happy to provide more insight here, if this is still unclear.
